schemas/database/003_unprocessed_identities.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE unprocessed_identities (
+    commitment    BYTEA       NOT NULL UNIQUE,
+    status        VARCHAR(50) NOT NULL,
+    created_at    TIMESTAMPTZ NOT NULL,
+    processed_at  TIMESTAMPTZ,
+    error_message TEXT
```

Comment:
Do we ever assign this error message?

Reply:
Now we do
https://github.com/worldcoin/signup-sequencer/pull/514/files#diff-23b4c110a535fa6df354073bab85cf1645400d08a9a202311db1d2898dc406bbR509
https://github.com/worldcoin/signup-sequencer/pull/514/files#diff-7f8025fc93b75296009eb2b41e0f99750760bf93b5115e03362e909036490109R70